src/aoapplication.cpp

```diff
@@ -18,6 +18,7 @@ AOApplication::AOApplication(int &argc, char **argv) : QApplication(argc, argv)
   discord = new AttorneyOnline::Discord();
   QObject::connect(net_manager, SIGNAL(ms_connect_finished(bool, bool)),
                    SLOT(ms_connect_finished(bool, bool)));
+  qApp->setStyleSheet("QFrame {background-color:transparent;} QAbstractItemView {background-color: transparent; color: black;}; QLineEdit {background-color:transparent;}");
```

Comment:
I think setting it for the whole app might cause problems.

Reply:
what could possibly go wrong
but ya legit it just sets the bg to transparent, there's no real bad implications.

Reply:
Dialogs

Reply:
@oldmud0 they work